src/theme.ts

```diff
@@ -49,6 +49,13 @@ export const theme = createTheme({
           color: common.white
         }
       }
+    },
+    MuiInputLabel: {
+      styleOverrides: {
+        root: {
+          color: common.black
+        }
+      }
```

Comment:
This change updated the main input on ambiance page
![Screenshot 2024-04-05 at 10 01 36 AM](https://github.com/hermanator608/ambiance/assets/29784945/cc74b58a-ddf5-4310-986f-546bf58a87f0)

Reply:
Honestly might be best to just create two different MUI Themes, would make styling much easier between the two pages. Then we can just swap the theme based on the url route?

Not part of this ticket, but maybe future update - can you create a task for this?